scripts/obtain/dotnet-install.sh

```diff
@@ -177,6 +180,9 @@ get_legacy_os_name() {
     elif [ -n "$runtime_id" ]; then
         echo $(get_legacy_os_name_from_platform "${runtime_id%-*}" || echo "${runtime_id%-*}")
         return 0
+    elif [[ $uname == MINGW* ]]; then
```

Comment:
For `CYGWIN` support, it would be:
`elif [[ $(uname) == MINGW* ]] || [[ $(uname) == CYGWIN* ]]; then`

Reply:
Yes I can add that if wanted, though this is only tested with git bash...

Reply:
Usually in windows Cygwin and MinGW both are considered in such introspections, they are two main distributions. See https://stackoverflow.com/a/3466183.

There is a portable (standalone) cygwin available for quick test: http://www.cybesystems.com/index.php?option=com_jdownloads&view=downloads&Itemid=196